nsv13/code/modules/clothing/custom_clothes.dm

```diff
@@ -18,8 +18,8 @@
 	can_adjust = TRUE
 
 /obj/item/clothing/under/ship/peacekeeper
-	name = "Peacekeeper uniform"
-	desc = "A padded jumpsuit worn exclusively by North Star peacekeeper forces. It's designed to be lightweight enough for patrols, but sturdy enough to keep you alive in CQB scenarios."
+	name = "Space cop uniform"
```

Comment:
I think names shouldn't start from uppercase, because it will look too notable.
He's wearing a Space cop uniform vs He's wearing a space cop uniform.